javaparser-core/src/main/javacc/java.jj

```diff
@@ -905,40 +917,40 @@ BodyDeclaration<?> ClassOrInterfaceBodyDeclaration():
    BodyDeclaration<?> ret;
 }
 {
-  (
-	  LOOKAHEAD(2)
-	  ret = InitializerDeclaration()
-	|
-      // Just get all the modifiers out of the way. If you want to do
-      // more checks, pass the modifiers down to the member
-	  modifier = Modifiers()
-	  (
-	      ret = ClassOrInterfaceDeclaration(modifier)
-	    | LOOKAHEAD("enum")
-	      ret = EnumDeclaration(modifier)
-	    | LOOKAHEAD("@" "interface")
-	      ret = AnnotationTypeDeclaration(modifier)
-	    | LOOKAHEAD( [ TypeParameters() ] Identifier() "(" )
-	      ret = ConstructorDeclaration(modifier)
-	    | LOOKAHEAD( Type() Identifier() ( ArrayBracketPair() )* ( "," | "=" | ";" ) )
-	      ret = FieldDeclaration(modifier)
-	    |
-	      ret = MethodDeclaration(modifier)
-	  )
-  )
-  { return ret; }
+    (
+        LOOKAHEAD(2)
+        ret = InitializerDeclaration()
+      |
+        // Just get all the modifiers out of the way. If you want to do
+        // more checks, pass the modifiers down to the member
+        modifier = Modifiers()
+        (
+            ret = ClassOrInterfaceDeclaration(modifier)
+          | LOOKAHEAD("enum")
+            ret = EnumDeclaration(modifier)
+          | LOOKAHEAD("@" "interface")
+            ret = AnnotationTypeDeclaration(modifier)
+          | LOOKAHEAD( [ TypeParameters() ] Identifier() "(" )
+            ret = ConstructorDeclaration(modifier)
+          | LOOKAHEAD( Type() Identifier() ( ArrayBracketPair() )* ("," | "=" | ";") )
+            ret = FieldDeclaration(modifier)
+          |
+            ret = MethodDeclaration(modifier)
+        )
+    )
+    { return ret; }
 }
 
 FieldDeclaration FieldDeclaration(ModifierHolder modifier):
 {
-	Type partialType;
-	NodeList<VariableDeclarator> variables = new NodeList<VariableDeclarator>();
-	VariableDeclarator val;
+    Type partialType;
+    NodeList<VariableDeclarator> variables = new NodeList<VariableDeclarator>();
+    VariableDeclarator val;
 }
 {
```

Comment:
A certain someone mashed the format-file key, it seems :-#

Reply:
... If only -- all manual I'm afraid :cry: 

I spent longer than probably sounds reasonable trying to figure out what's what -- sorting out spacing and indentation as I went :D 

Happy to revert these unrelated changes now that I understand the difference between ()/[]/{} blocks and the structure :+1:

Reply:
Make it as you like it best :-) It just took me some time to find the grammar changes in the github diff. And I'm aware the formatting is horrible in the grammar, so good job

Reply:
For stuff like this I tend to really appreciate the intellij diffs -- there you can set it to ignore whitespace changes.

If you want to see git blame, right click on the gutter with the lite numbers and use "annotate". That can be configured to also ignore whitespace only changes and even moves within the file.

That's been handy a few times in trying to figure out which commit made a particular addition/change, avoiding the noise of whitespace only changes.